### PR TITLE
[Merged by Bors] - Subject Matter Experts and new Bevy Org docs

### PR DESCRIPTION
**Pull request by cart**
We are in the process of rolling out a new Bevy Organization role! (Subject Matter Expert)

This adds a new "The Bevy Organization" document and links to it from CONTRIBUTING.md. This doc describes how the Bevy Organization will work going forward. It outlines the functionality of each role, as well as the expectations we have for them. The previously existing roles (Project Lead, Maintainer) still work the same way, but their definition and scope have been made much clearer.

Tomorrow we will be announcing this publicly in a blog post. This will describe the motivation and announce the first round of SMEs . But before that  it makes sense to do a quick review round first.

Given the quick turnaround on this PR, this isn't the best platform to discuss changes to the SME system (or its validity). After you have read the announcement tomorrow, feel free to start discussions wherever is preferable to you (this repo, discord, etc). So for now, please just review for clarity / typos / phrasing / missed info / etc.

[Rendered](https://github.com/bevyengine/bevy/blob/08ceae43dbfacf630447e5f5f1cfa53397242687/docs/the_bevy_organization.md)

### CODE REVIEW

**Comment by IceSentry** [comment]
Do you have a rendered link?

File: docs/the_bevy_organization.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Bevy Org Member / Triage Team
+
+[Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who:
+
+1. Have actively engaged with Bevy development.
+2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
+3. Have asked to join the Bevy Org. Reach out to @cart on [Discord] or email us at bevyengine@gmail.com if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
+
+All Bevy Org members are also Triage Team members. The Triage Team can label and close issues and PRs but do not have merge rights or any special authority within the community.
```
Comment (IceSentry) [inline docs/the_bevy_organization.md:72]: This could also mention that org members are allowed to do `bors try`

**Comment by cart** [comment]
> Do you have a rendered link?
Good call. Just added one to the description.

File: docs/the_bevy_organization.md (hunk 1 above)
Comment (cart) [reply docs/the_bevy_organization.md:72]: Good call!

Comment (alice-i-cecile) [reply docs/the_bevy_organization.md:72]:
```suggestion
All Bevy Org members are also Triage Team members. The Triage Team can label and close issues and PRs but do not have merge rights or any special authority within the community.
Org members also have the ability to use `bors try`, causing our build system to evaluate the PR again.
```

Comment (mockersf) [reply docs/the_bevy_organization.md:72]: Bors also run additional jobs that aren't run by normal ci: iOS, Android, windows, no default features

File: docs/the_bevy_organization.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+(Finalizing this list now)
+
+### Bevy Org Member / Triage Team
+
+[Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who:
+
+1. Have actively engaged with Bevy development.
+2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
+3. Have asked to join the Bevy Org. Reach out to @cart on [Discord] or email us at bevyengine@gmail.com if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
```
Comment (mockersf) [inline docs/the_bevy_organization.md:70]: Missing the link for discord

File: docs/the_bevy_organization.md (hunk 1 above)
Comment (alice-i-cecile) [reply docs/the_bevy_organization.md:72]: Updated, thanks.

File: docs/the_bevy_organization.md (hunk 2 above)
Comment (alice-i-cecile) [reply docs/the_bevy_organization.md:70]:
```suggestion
3. Have asked to join the Bevy Org. Reach out to @cart on [Discord](https://disord.gg/bevy) or email us at bevyengine@gmail.com if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
```

File: docs/the_bevy_organization.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+## The Bevy Organization
```
Comment (alice-i-cecile) [inline docs/the_bevy_organization.md:1]: All your headings are off by one level.

Comment (cart) [reply docs/the_bevy_organization.md]: Ooh good catch!

**Pull request closed without merging**